front/app/containers/Admin/projects/project/projectHeader/ReviewFlow/index.tsx

```diff
@@ -57,21 +46,22 @@ const ReviewFlow = ({ project }: { project: IProjectData }) => {
   }
 
   const approvalPending = projectReview?.data.attributes.state === 'pending';
-  const approvalGranted = projectReview?.data.attributes.state === 'approved';
-
-  const showProjectApprovalButton = approvalPending && canReview;
-  const showReviewRequestButton =
-    isProjectReviewEnabled && !canReview && !approvalGranted;
-  const showPublishButton =
-    !showProjectApprovalButton && !showReviewRequestButton;
+  const showReviewRequestButton = isProjectReviewEnabled && !canPublish;
+  const showProjectApprovalButton =
+    isProjectReviewEnabled && canPublish && approvalPending;
+  // showPublishButton could in practice only result in false if the
+  // project_review feature is enabled and the user is a project or folder manager trying to publish at the root level
+  const showPublishButton = !showReviewRequestButton;
```

Comment:
Think it's okay for the publish button to show even if there's also an "Approve" button next to it. Then admins have the option to publish directly instead of only approving. Looks clear enough to me. Wdyt?

<img width="589" alt="image" src="https://github.com/user-attachments/assets/c06a4bbb-8cbf-47fe-9059-1782845e44b9" />

Reply:
Agreed! will rework that part.